app/views/content/is-teaching-right-for-me/behaviour-management.md

```diff
@@ -0,0 +1,95 @@
+
+## What support is there for trainee teachers?
+As a teacher you’re never going to be perfect. You’re constantly learning and the variety and support you get through your day through learning all of these different things is really important.
+
+If you have to pay a fee to do your teacher training course and it leads to qualified teacher status (QTS) you might see lots of different classes and how a particular teacher works with them. Or you might go round with a particular student and see what a day in their lives looks like, with the different teachers they may have that day. This allows you to pick up strategies other people may use while also seeing what kind of challenges a particular class or  child might have during that period of time.
+
+Then you might work either with small groups or one-to-one with pupils. Again you're observing and analysing what's working. You're not alone. You've got a teacher there so if you think something didn't quite land you've got somebody to bounce ideas off and look at how you might change it.
+
+## Once I become a teacher is there any support available?
+All schools have behaviour policies in place, so you work within a remit. If your course leads to QTS, for the first two years of teaching as an [early career teacher (ECT)](/support-for-early-career-teachers) you'll have a named mentor. But you've also got other colleagues and it's not unheard of for somebody to sit down and say they're having a problem with child A what are you doing to keep them in line? You're constantly working together to swap ideas, sanctions and bits of information about that child. 
+
+If you want to develop your behaviour management skills there are [specialist national professional qualification (NPQ) courses](https://www.gov.uk/guidance/national-professional-qualification-npq-courses) available to help teachers learn how to create a culture of good behaviour in schools. 
+
+## How can I remember children's names?
+Knowing a child’s name is a really powerful behaviour management tool, so learning names is essential for building strong relationships.
+
+Making sure pupils put their names on the top of their page provides a helpful prompt for you. You can also make a seating plan and keep that handy. Doing an ice-breaker activity with new classes where pupils say their names and something that is unique to them is a great technique too.
+
+Giving them nicknames can also be a good way to help you remember and it makes the children feel special too.
+
+## If I'm a quiet person will I be assertive enough?
+
+Your teacher training course will allow you to experiment with the best tone of voice for your own teaching style. Observe as many teachers as you can and cherry-pick the techniques and skills that they use for your own classroom.
```

Comment:
```suggestion
Your teacher training course will allow you to experiment with the best tone of voice for your own teaching style. Observe as many teachers as you can and cherrypick the techniques and skills that they use for your own classroom.
```

Reply:
@gemmadallmandfe this is from the original blog post. I checked online and the consensus seems to be cherry-pick is hyphenated. https://www.oed.com/dictionary/cherry-pick_v?tl=true https://dictionary.cambridge.org/dictionary/english/cherry-pick